js/.jshintrc

```diff
@@ -0,0 +1,98 @@
+	"multistr"      : false,    // Tolerate multi-line strings.
+	"onecase"       : false,    // Tolerate switches with just one case.
+	"proto"         : false,    // Tolerate __proto__ property. This property is deprecated.
+	"regexdash"     : false,    // Tolerate unescaped last dash i.e. `[-...]`.
+	"scripturl"     : false,    // Tolerate script-targeted URLs.
+	"smarttabs"     : false,    // Tolerate mixed tabs and spaces when the latter are used for alignmnent only.
+	"shadow"        : false,    // Allows re-define variables later in code e.g. `var x=1; x=2;`.
+	"sub"           : false,    // Tolerate all forms of subscript notation besides dot notation e.g. `dict['key']` instead of `dict.key`.
+	"supernew"      : false,    // Tolerate `new function () { ... };` and `new Object;`.
+	"validthis"     : false,    // Tolerate strict violations when the code is running in strict mode and you use this in a non-constructor function.
+
+	// == Environments ====================================================
+
+	"browser"       : true,     // Standard browser globals e.g. `window`, `document`.
+	"couch"         : false,    // Enable globals exposed by CouchDB.
+	"devel"         : false,    // Allow development statements e.g. `console.log();`.
+	"dojo"          : false,    // Enable globals exposed by Dojo Toolkit.
+	"jquery"        : true ,    // Enable globals exposed by jQuery JavaScript library.
+	"mootools"      : false,    // Enable globals exposed by MooTools JavaScript framework.
+	"node"          : true,     // Enable globals available when code is running inside of the NodeJS runtime environment.
+	"nonstandard"   : false,    // Define non-standard but widely adopted globals such as escape and unescape.
+	"prototypejs"   : false,    // Enable globals exposed by Prototype JavaScript framework.
+	"rhino"         : false,    // Enable globals available when your code is running inside of the Rhino runtime environment.
+	"wsh"           : false,    // Enable globals available when your code is running as a script for the Windows Script Host.
+
+	// == JSLint Legacy ===================================================
+
+	"nomen"         : false,    // Prohibit use of initial or trailing underbars in names.
+	"onevar"        : false,    // Allow only one `var` statement per function.
+	"passfail"      : false,    // Stop on first error.
+	"white"         : false,    // Check against strict whitespace and indentation rules.
+
+	// == Undocumented Options ============================================
+
+	"maxerr"        : 100,      // Maximum errors before stopping.
+	"globals"       : {         // Extra globals.
+		"Path": false,
+		"WebFrame": false,
+		"IPC": false,
+		"ElectronScreen": false,
+		"RendererIPC": false,
+		"Fs": false,
+		"Shell": false,
+		"Process": false,
+		"Remote": false,
+		"mainWindow": true,
+		"UI": true,
+		"Daemon": true,
+		"Plugins": true
+	},
+	"indent"        : 4         // Specify indentation spacing
+}
```

Comment:
This is so index.js can be linted with different globals than the files in the js folder.

Also Files, Hosting, and Wallet all have their own .jshintrc's as well.